src/app/images/components/ImagesTable/ImagesTable.tsx

```diff
@@ -149,7 +149,9 @@ const generateResourceRow = ({
   const { os, release } = splitResourceName(resource.name);
   const isCommissioningImage =
     os === "ubuntu" && release === commissioningRelease;
-  const canBeDeleted = !isCommissioningImage && resource.complete;
+  const canBeDeleted =
+    (!isCommissioningImage && resource.complete) ||
+    (!isCommissioningImage && !resource.downloading);
```

Comment:
I'd suggest to simplify.
```suggestion
const canBeDeleted = !isCommissioningImage && (resource.complete || !resource.downloading);
```